library/cmac.c

```diff
@@ -779,6 +779,8 @@ static int cmac_test_subkeys( int verbose,
 
         if( verbose != 0 )
             mbedtls_printf( "passed\n" );
+
+        mbedtls_cipher_free( &ctx );
```

Comment:
No, because we allocate it _inside_ the `for` loop in line 747 with a `mbedtls_cipher_setup()`. (The `mbedtls_cipher_init()` function just fills the structure with zeroes and does not allocate the buffers)